src/_gettsim_tests/test_renten_alter.py

```diff
@@ -0,0 +1,110 @@
+import itertools
+import random
+
+import pandas as pd
+import pytest
+from _gettsim.interface import compute_taxes_and_transfers
+from _gettsim.policy_environment import set_up_policy_environment
+from _gettsim_tests import TEST_DATA_DIR
+from pandas.testing import assert_series_equal
+
+
+INPUT_COLS = [
+    "p_id",
+    "hh_id",
+    "tu_id",
+    "alter",
+    "jahr",
+    "geburtsjahr",
+    "geburtsmonat",
+    "m_arbeitsunfähig",
+    "m_krank_ab_16_bis_24",
+    "m_mutterschutz",
+    "m_arbeitslos",
+    "m_ausbild_suche",
+    "m_schul_ausbild",
+    "m_alg1_übergang",
+    "m_geringf_beschäft",
+    "weiblich",
+    "y_pflichtbeitr_ab_40",
+    "m_pflichtbeitrag",
+    "m_freiw_beitrag",
+    "m_ersatzzeit",
+    "m_kind_berücks_zeit",
+    "m_pfleg_berücks_zeit",
+]
+
+OUT_COLS = [
+    "_ges_rente_altersgrenze_abschlagsfrei",
+]
+
+YEARS = [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021]
+
+
+@pytest.fixture(scope="module")
+def input_data():
+    # Original Data Source: Seibold, A. (2021).
+    # Reference points for retirement behavior:
+    # Evidence from german pension discontinuities.
+    # American Economic Review, 111(4), 1126-65.
```

Comment:
We should mention the exact table here. Is it in the online appendix? WOuld be also interested to respond to the other question.

Reply:
Thank you! Should add that here.